atomic_reactor/utils/operator.py

```diff
@@ -23,6 +24,181 @@
 
 OPERATOR_CSV_KIND = "ClusterServiceVersion"
 
+# Schema derived from https://github.com/operator-framework/api/blob/e29d40c2eb3eab6a2789a671886b8966ce8e3c0a/crds/operators.coreos.com_clusterserviceversions.yaml#L40  # noqa E501
+_mini_csv_schema = {
```

Comment:
Any reason not put this schema into the `schemas/`?

Reply:
@MartinBasti? -^

Reply:
https://github.com/containerbuildsystem/operator-manifest  this code will be synced to shared library. Eventually, there will be only shared library for operator manipulations in future

Reply:
@tkdchen, @mkosiarc; tldr; Maintaining *one* file in common between this repo and [operator_manifest](https://github.com/containerbuildsystem/operator-manifest) is easier than maintaining more than one.

This is a temporary situation, i.e. until operator_manifest truly becomes its own thing.